apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -184,12 +184,19 @@ <h3 class="ledgerHead">
                     </thead>
                     <tbody>
                         <tr>
-                            <td class="nopad bdrL bdrR hidden-xs" style="width:50%" colspan="3">
+                            <td width="100%" colspan="100%" class="ledger-head transction-mapped">
+                                <table class="table" width="100%">
+                                    <tr>
+                                        <th colspan="6">Debit <span class="pull-right text-gray">Dr</span></th>
```

Comment:
Please check if new text is present in JSON else add it

Reply:
Added     **"app_dr": "dr."** by this name